cashier-paddle.md

```diff
@@ -0,0 +1,859 @@
+### Paddle JS
+
+Paddle relies on its own JavaScript library to initiate the checkout widget. You can load it by placing the `@paddleJS` directive right before the closing `</head>` tag:
+
+        @paddleJS
+    </head>
+    <body>
+
+<a name="currency-configuration"></a>
+### Currency Configuration
+
+The default Cashier currency is United States Dollars (USD). You can change the default currency by setting the `CASHIER_CURRENCY` environment variable:
+
+    CASHIER_CURRENCY=EUR
+
+In addition to configuring Cashier's currency, you may also specify a locale to be used when formatting money values for display on invoices. Internally, Cashier utilizes [PHP's `NumberFormatter` class](https://www.php.net/manual/en/class.numberformatter.php) to set the currency locale:
+
+    CASHIER_CURRENCY_LOCALE=nl_BE
+
+> {note} In order to use locales other than `en`, ensure the `ext-intl` PHP extension is installed and configured on your server.
+
+<a name="core-concepts"></a>
+## Core Concepts
+
+<a name="pay-links"></a>
+### Pay Links
+
+One of the important things to know is that Paddle lacks an extensive CRUD API to perform state changes. Therefore, most of the Paddle interaction is done through [its checkout widget](https://developer.paddle.com/guides/how-tos/checkout/paddle-checkout). To allow us to generate this widget easily we use Paddle's API to generate pay links:
+
+    $user = User::find(1);
+
+    $payLink = $user->newSubscription('default', 'premium')
+        ->returnTo(route('home'))
+        ->create();
+
+We then use these pay links on a `paddle-button` Blade component that ships with Cashier:
+
+    <x-paddle-button :url="$payLink" class="w-8 h-4">
+        Subscribe
+    </x-paddle-button>
+
+Because this checkout widget works asynchronously, creation and updating of subscriptions is done through webhooks. Usually the delay for this is minimal but you should account for this in your app by considering that your user's subscription might not be immediately available after completing the checkout. It's important that you properly [set up webhooks](#handling-paddle-webhooks) to accommodate for state changes from Paddle.
+
+You can check out [the API documentation on pay link generation](https://developer.paddle.com/api-reference/product-api/pay-links/createpaylink) for more info.
+
+<a name="inline-checkout"></a>
+### Inline Checkout
+
+If you don't want to make use of the overlay checkout widget, Paddle also has an option to display an inline checkout. While you cannot properly adjust any of HTML fields (it's basically the checkout widget but inline) it allows you to embed it within your app's look and feel.
+
+To make it easy for you to get started with inline checkout we've shipped a `paddle-checkout` Blade component for you to use. You can generate a pay link just like you do with the overlay widget and pass that to the component's `override` attribute:
+
+    <x-paddle-checkout :override="$payLink" class="w-full" />
```

Comment:
What is the meaning of "override" here?

Reply:
It conforms to the option "override" on checkout to basically set up the checkout widget using a pay link.

https://developer.paddle.com/reference/paddle-js/parameters#checkout-properties

<img width="1503" alt="Screenshot 2020-06-02 at 17 03 32" src="https://user-images.githubusercontent.com/594614/83536107-1b490580-a4f3-11ea-8302-60bfce253543.png">